ACD-618 (#30)
Actindo-AG/Documentation: Channels/Integration/01f_Tradebyte.md

5. Enter the shop name in the *Shop name* field.

6. Enter the username and the password for API access in the corresponding *Username (REST)* and *Password (REST)* fields.

7. Enter the username and the password for server access in the corresponding *Username (SFTP)* and *Password (SFTP)* fields.

8. If desired, enable the *Test API?* toggle if the API to be connected corresponds to a test account. Disable the toggle if the API to be connected corresponds to a productive account. By default, the toggle is disabled.

9. Click the [SAVE] button.  
    The *Checking credentials* notice is displayed. The connection will be established and the synchronization is triggered. The *Sync triggered* pop-up window is displayed.  

    ![Sync triggered](../../Assets/Screenshots/Channels/Settings/Connections/SyncTriggered.png "[Sync triggered]")

    The *CONNECTIONS* tab in the *Settings* menu entry of the *Omni-Channel* module is displayed when the connection has been established. The *Tradebyte* connection is displayed in the list of connections.

    ![Tradebyte connection](../../Assets/Screenshots/Channels/Settings/Connections/Tradebyte/Connections.png "[Tradebyte connection]")	

> 10. If necessary, continue to configure the Tradebyte connection, see [Configure the Tradebyte connection](#configure-the-Tradebyte-connection).

Reviewer: ```suggestion
10. If necessary, continue to configure the Tradebyte connection after the initial sync, see [Configure the Tradebyte connection](#configure-the-Tradebyte-connection).
```

## Configure the Tradebyte connection   

After the connection to a Tradebyte shop has been established, further settings can be configured for the connection.

#### Prerequisites

> A Tradebyte connection has been established.

Reviewer: ```suggestion
A Tradebyte connection has been established and synchronised.
```

# Manage the Tradebyte connection

To establish a connection to a Tradebyte shop, there are several particularities to consider. Therefore, the configuration of the Tradebyte connection is described in detail below.

> 

Author: Warum nicht die creation?

## Configure the Tradebyte connection   

After the connection to a Tradebyte shop has been established, further settings can be configured for the connection.

#### Prerequisites

> - A Tradebyte account has been created.
> - The *Tradebyte Integration* plugin has been installed.
> - A Tradebyte connection has been established.

Author: ```suggestion
A Tradebyte connection has been established.
```

> [Info] For the *Omni-Channel* module version 4.1.0 or higher, the *Tradebyte Integration* plugin is required in at least version 1.0.0.

[comment]: <> (Stimmt das so? In Platform Manager Tradebyte 1.0.0 bis 1.2.0 verfügbar.)

#### Procedure

*Omni-Channel > Settings > Tab CONNECTIONS*

![Tradebyte connection](../../Assets/Screenshots/Channels/Settings/Connections/Tradebyte/Connection.png "[Tradebyte connection]")

1. Click the Tradebyte connection in the list of connections.   

>   The *Edit connection* view is displayed. By default, the *Credentials* tab is displayed.

Author: ```suggestion
  The *Edit connection* view is displayed. By default, the *Credentials* tab is selected.
```

  ![Edit connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/Tradebyte/EditConnectionCredentials.png "[Edit connection credentials]")

2. Click the *Settings* tab.   

>   The *Settings* tab is displayed. By default, the *Selected sales channels* setting is displayed. All sales channels available in Tradebyte are displayed in the right side bar. 

Author: ```suggestion
  The *Settings* tab is displayed. By default, the *Selected sales channels* setting is selected.
```

  ![Edit connection settings](../../Assets/Screenshots/Channels/Settings/Connections/Tradebyte/EditConnectionSettings.png "[Edit connection settings]")

> 3. Select the checkbox of the sales channels you want to use.  

Author: ```suggestion
3. Select the checkboxes of the sales channels you want to activate.  
```

4. Click the *Enable languages* menu entry in the left side bar.  
    All languages available in Tradebyte are displayed in the right side bar. 

> 5. Enable the toggles of the languages you want to use.

Author: ```suggestion
5. Enable the toggles of the languages you want to maintain in the *Actindo Core1 Platform".
```

    > [Info] The default language is automatically set and cannot be edited.

6. Click the [Save] button.  
    All changes have been saved. The *Saving successful* pop-up window is displayed

    ![Saving successful](../../Assets/Screenshots/Channels/Settings/Connections/SavingSuccessful.png "[Saving successful]")

> [comment]: <> (Evtl. noch Header Configure the Magento2 ETL mapping hinzufügen mit der Erklärung Tradebyte driver uses the standard ETL mapping. Therefore, there are no further specific settings/aspects to be considered... oder sowas. Gleiches gilt für Magento.)

Author: ```suggestion
```
Standard muss nicht weiter beschrieben werden.
